### PR TITLE
[stable/selenium] - Adding the option to set nodePort on the service

### PR DESCRIPTION
**Pull request by ekirmayer**
Signed-off-by: Elad Kirmayer <ekirmayer@hotmail.com>

#### What this PR does / why we need it:

This pull request is for allowing to select the nodePort on which the selenium-hub will be exposed with
This will make it simpler when deploying without the need to add queries to get the exposed port number. needed 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/selenium/templates/hub-service.yaml [hunk 1]
```diff
@@ -25,5 +25,8 @@ spec:
   - name: hub
     port: {{ .Values.hub.servicePort }}
     targetPort: {{ .Values.hub.port }}
+    {{- if .Values.hub.nodePort }}
```
Comment (scottrigby) [inline stable/selenium/templates/hub-service.yaml]: We should also check if `.Values.hub.serviceType` is `NodePort` before rendering this part of the service spec.

Comment (ekirmayer) [reply stable/selenium/templates/hub-service.yaml]: I pushed the chart with this tests combination

File: stable/selenium/.helmignore [hunk 2]
```diff
@@ -19,3 +19,6 @@
 .project
 .idea/
 *.tmproj
+
+# OWNERS file for Kubernetes
+OWNERS
```
Comment (scottrigby) [inline stable/selenium/.helmignore]: Needs newline at EOF

File: stable/selenium/OWNERS [hunk 3]
```diff
@@ -0,0 +1,8 @@
+approvers:
+- flah00
+- diemol
+- ekirmayer
+reviewers:
+- flah00
+- diemol
+- ekirmayer
```
Comment (scottrigby) [inline stable/selenium/OWNERS]: Needs newline at EOF

File: stable/selenium/OWNERS [hunk 4]
```diff
@@ -0,0 +1,8 @@
+approvers:
+- flah00
```
Comment (scottrigby) [inline stable/selenium/OWNERS:2]: @ekirmayer can you check with these users to see if they want to be in the OWNERS file for this chart?

Comment (ekirmayer) [reply stable/selenium/OWNERS]: /cc @flah00

Comment (ekirmayer) [reply stable/selenium/OWNERS:2]: @scottrigby - I only managed to get @flah00. I think we should keep both of them in the OWNERS file.

Comment (diemol) [reply stable/selenium/OWNERS:2]: Hi, just saw your message @ekirmayer, I'd like to be added as well.

File: stable/selenium/Chart.yaml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 name: selenium
-version: 1.0.3
+version: 1.0.4
 appVersion: 3.14.0
```
Comment (diemol) [inline stable/selenium/Chart.yaml:3]: Since you are updating the chart, @ekirmayer why don't you upgrade to the latest version? 3.141.59?

Comment (diemol) [reply stable/selenium/Chart.yaml:3]: https://github.com/helm/charts/blob/master/stable/selenium/values.yaml#L21

**Comment by flah00** [comment]
Lgtm

File: stable/selenium/Chart.yaml (hunk 5 above)
Comment (ekirmayer) [reply stable/selenium/Chart.yaml:3]: @diemol - I thought about it, but don’t you think we should split updating the version from adding features?

Comment (diemol) [reply stable/selenium/Chart.yaml:3]: You are right @ekirmayer, better to do two separate PRs. 👍

Comment (ekirmayer) [reply stable/selenium/Chart.yaml:3]: @diemol - Can you approve the this for merge?

**Comment by diemol** [comment]
lgtm

**Review by diemol — APPROVED** [review]

**Review by scottrigby — APPROVED** [review]

**Comment by scottrigby** [comment]
Thanks for your work on this @ekirmayer and @diemol and @flah00 for reviewing. After this is merged, you will be able to review/approve PRs for this chart.

**Pull request merged**